osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -96,6 +96,7 @@ protected override void LoadComplete()
         {
             updateTooltipText(Current.Value);
```

Comment:
You can remove this line because `BindValueChanged` with second argument `true` invokes this method (which is what peppy said)